ci/check_format.sh

```diff
@@ -25,12 +25,16 @@ set -euf -o pipefail
 OUT_DIR="/tmp/parsec-book-ci"
 mkdir -p $OUT_DIR
 
+# Add the node modules .bin directory to the path for `markdown-link-check`
+export PATH=$PATH:$(pwd)/node_modules/.bin
```

Comment:
Maybe it will be better for that `export` to be in the CI workflow and not here so that it would not change the user's `PATH` variable if he executes it locally. Same than what is there for the Go binaries.

Reply:
indeed, was not keen on doing it this way - before I put it in the task before the check_format for some reason, but didn't think of putting it in the same task 🤕